cfme/tests/middleware/jdr_report_methods.py

```diff
@@ -1,29 +1,32 @@
 from cfme.utils.wait import wait_for
 
+DELAY = 30
```

Comment:
Same as comments above, would like to hear justification for the increase in delay and timeout.